# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/test_proxyconnect.js
# Commit: ee9744982673
# Full Hash: ee974498267381418f06efd6ab92c1f2c5f3735d
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-15 16:44:51
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r f5a4bfdaba40 -r ee9744982673 netwerk/test/unit/test_proxyconnect.js
--- a/netwerk/test/unit/test_proxyconnect.js	Tue Nov 15 05:34:40 2022 +0000
+++ b/netwerk/test/unit/test_proxyconnect.js	Tue Nov 15 05:34:40 2022 +0000
@@ -255,7 +255,9 @@
     streamIn = transport
       .openInputStream(0, SEGMENT_SIZE, SEGMENT_COUNT)
       .QueryInterface(Ci.nsIAsyncInputStream);
-    streamOut = transport.openOutputStream(0, 0, 0);
+    streamOut = transport
+      .openOutputStream(0, 0, 0)
+      .QueryInterface(Ci.nsIAsyncOutputStream);
 
     streamIn.asyncWait(connectHandler, 0, 0, threadManager.mainThread);
   } catch (e) {